DNN Platform/Library/Services/FileSystem/FileInfo.cs

```diff
@@ -69,18 +69,18 @@ public FileInfo()
             VersionGuid = Guid.NewGuid();
         }
 
-        public FileInfo(int portalId, string filename, string extension, int filesize, int width, int height, string contentType, string folder, int folderId, int storageLocation, bool cached)
+        public FileInfo(int portalId, string filename, string extension, long filesize, int width, int height, string contentType, string folder, int folderId, int storageLocation, bool cached)
             : this(portalId, filename, extension, filesize, width, height, contentType, folder, folderId, storageLocation, cached, Null.NullString)
         {
         }
 
-        public FileInfo(int portalId, string filename, string extension, int filesize, int width, int height, string contentType, string folder, int folderId, int storageLocation, bool cached,
+        public FileInfo(int portalId, string filename, string extension, long filesize, int width, int height, string contentType, string folder, int folderId, int storageLocation, bool cached,
```

Comment:
This creates backward compatibility issue. Must deprecate and create another one.